filebeat/docs/fields.asciidoc

```diff
@@ -52955,6 +52955,13 @@ type: keyword
 
 --
 
+*`tls.server.server_name`*::
```

Comment:
I'm not sure about adding this field it isn't in ECS.  The tls.client.server_name is usually used for the SNI, so I'm not sure what tls.server.server_name would map to.  In the example logs I think the data that is being mapped is the server certificate common name.

Reply:
I am okay removing that one. Kept it because was already mapped and new checks triggered the no doc alert one re generating the golden files. I wonder if it would be a breaking change, but since it was wrongly mapped before anyway (mapping to client when its sourced from a server field) maybe is not a big issue, so not sure what would be best in this case.

Reply:
I think because it was mapped wrong we should fix and document in the bugfix section.